const.go

```diff
@@ -0,0 +1,73 @@
+package paypal
+
+type SubscriptionPlanStatus string
+
+const (
+	SUBSCRIPTION_PLAN_STATUS_CREATED  SubscriptionPlanStatus = "CREATED"
+	SUBSCRIPTION_PLAN_STATUS_INACTIVE SubscriptionPlanStatus = "INACTIVE"
+	SUBSCRIPTION_PLAN_STATUS_ACTIVE   SubscriptionPlanStatus = "ACTIVE"
+)
+
+type IntervalUnit string
+
+const (
+	INTERVAL_UNIT_DAY   IntervalUnit = "DAY"
```

Comment:
super tiny nit:

we are mixing camel cases and partial snake cases thru this file.

wdyt about the following format

```
IntervalUnitWeek  IntervalUnit = "WEEK"
```

similarly 

```
ShippingPreferenceGetFromFile instead of ShippingPreference_GET_FROM_FILE
```